sydent/sydent.py

```diff
@@ -174,6 +174,18 @@ def set_from_comma_sep_string(rawstr):
         # The number of characters from the beginning to reveal of the email's domain
         # portion (right of the '@' sign)
         'email.third_party_invite_domain_obfuscate_characters': '3',
+
+        # A string to separate multiple components of the username portion of an email address.
+        # For instance, if "." is set, then "alice.smith@example.com" would result
+        # in both "alice" and "smith" being individually obfuscated. Resulting in
+        # "ali...smi...@example.com" for example.
```

Comment:
This example seems wrong, wouldn't it end up being `ali....smi...@example.com`, note the four dots after `ali` (you end up with `...` from the redaction and then one from the separator).

It seems like it would have been nicer to use the ellipses character (`…`) instead of 3 dots (`...`) here.

Reply:
Oh, using `…` is an excellent idea! Yes, with the ` if component != ""` check I was trying to eliminate the edge case of having four dots, but I think this might be a more elegant solution.

Of course doing so is going to conflict with each of the other PRs, but the change itself won't have any backwards compatibility concerns.

I might leave the edge case of four dots in this PR, and make another one when all 3 PRs are merged to replace `...` with the ellipses.

Reply:
I've also changed the example to use `-`, which is clearer.